src/Logging/Logging.File/Extensions/LogService.File.Extensions.cs

```diff
@@ -0,0 +1,33 @@
+﻿using CodeMonkeys.Logging.File;
+
+namespace CodeMonkeys.Logging
+{
+    public static partial class LogServiceExtensions
+    {
+        /// <summary>
+        /// If a <see cref="FileLogService"/> is attached to the <see cref="LogService"/> this method enables
+        /// logging with it.
+        /// </summary>
+        public static void EnableFileLogging(this ILogService @this)
+        {
+            Argument.NotNull(
+                @this,
+                nameof(@this));
+
+            @this.EnableScopedService<FileLogService>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'this' before using it.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5626772)